projects/observability/src/shared/components/entity-renderer/entity-renderer.component.ts

```diff
@@ -96,7 +96,7 @@ export class EntityRendererComponent implements OnChanges {
 
   private setNavigationParams(): void {
     this.navigationParams =
-      this.navigable && this.entity !== undefined
+      this.navigable && this.entity !== undefined && this.entity[entityIdKey] !== 'null'
```

Comment:
Why are we passing invalid entities?

Reply:
we can have attacks identified on requests for which we haven’t learnt the API endpoints. For that events we get apiId = null and `apiName = null`